CDP4Dal/Session.cs

```diff
@@ -948,33 +948,16 @@ private void AddIterationToOpenList(Guid iterationId, DomainOfExpertise activeDo
             this.AddRdlToOpenList(modelRdl);
         }
 
-        /// <summary>
-        /// Queries the current <see cref="DomainOfExpertise"/> from the session for the current <see cref="Iteration"/>
-        /// </summary>
-        /// <returns>
-        /// The <see cref="DomainOfExpertise"/> if selected, null otherwise.
-        /// </returns>
-        public DomainOfExpertise QueryCurrentDomainOfExpertise()

```

Comment:
i think there was indeed already a method called `QuerySelectedDomainOfExpertise(Iteration iteration)` that was taking care of this, and indeed, it needs an `Iteration` as parameter